package.json

```diff
@@ -37,53 +37,62 @@
   "dependencies": {
     "@angular-extensions/elements": "^8.4.1",
     "@angular-extensions/model": "^8.0.0",
-    "@angular/animations": "~8.0.0",
-    "@angular/cdk": "~8.0.0",
-    "@angular/common": "~8.0.0",
-    "@angular/compiler": "~8.0.0",
-    "@angular/core": "~8.0.0",
-    "@angular/forms": "~8.0.0",
-    "@angular/http": "^8.0.0-beta.10",
-    "@angular/material": "~8.0.0",
-    "@angular/platform-browser": "~8.0.0",
-    "@angular/platform-browser-dynamic": "~8.0.0",
-    "@angular/router": "~8.0.0",
+    "@angular/animations": "^8.1.0",
+    "@angular/cdk": "^8.1.0",
+    "@angular/common": "^8.1.0",
+    "@angular/compiler": "^8.1.0",
+    "@angular/core": "^8.1.0",
+    "@angular/fire": "^5.2.1",
+    "@angular/forms": "^8.1.0",
+    "@angular/material": "^8.1.0",
+    "@angular/material-moment-adapter": "^8.1.0",
+    "@angular/platform-browser": "^8.1.0",
+    "@angular/platform-browser-dynamic": "^8.1.0",
+    "@angular/router": "^8.1.0",
     "@fortawesome/angular-fontawesome": "^0.4.0",
     "@fortawesome/fontawesome-free": "^5.8.2",
     "@fortawesome/fontawesome-svg-core": "^1.2.18",
     "@fortawesome/free-brands-svg-icons": "^5.8.2",
     "@fortawesome/free-solid-svg-icons": "^5.8.2",
-    "@ngrx/effects": "^8.0.0",
-    "@ngrx/entity": "^8.0.0",
-    "@ngrx/router-store": "^8.0.0",
-    "@ngrx/store": "^8.0.0",
-    "@ngrx/store-devtools": "^8.0.0",
+    "@ngrx/effects": "^8.1.0",
+    "@ngrx/entity": "^8.1.0",
+    "@ngrx/router-store": "^8.1.0",
+    "@ngrx/store": "^8.1.0",
+    "@ngrx/store-devtools": "^8.1.0",
     "@ngx-translate/core": "^11.0.1",
     "@ngx-translate/http-loader": "^4.0.0",
     "bootstrap": "^4.3.1",
     "browser-detect": "^0.2.28",
+    "firebase": "^6.3.5",

```

Comment:
Don't see any reason to add dependencies while not using them, also firebase is currently out of scope of this project, it is meant to be beginners friendly and is already pretty complex...

Also store freeze is not needed with ngrx 8 so please remove all the extra added dependencies...